main.go

```diff
@@ -2,8 +2,8 @@ package main
 
 import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-	//"github.com/jfrog/terraform-provider-xray/pkg/xray"
-	"terraform-provider-xray/pkg/xray"
+	"github.com/jfrog/terraform-provider-xray/pkg/xray"
+	//"terraform-provider-xray/pkg/xray"
```

Comment:
??

Reply:
same, added to make provider work locally, removing